src/compile/axis/properties.ts

```diff
@@ -67,9 +67,12 @@ export function orient(channel: PositionScaleChannel) {
   throw new Error(log.message.INVALID_CHANNEL_FOR_AXIS);
 }
 
-export function tickCount(channel: PositionScaleChannel, fieldDef: FieldDef<string>, scaleType: ScaleType, size: VgSignalRef) {
+export function tickCount(specifiedAxis: Axis, channel: PositionScaleChannel, fieldDef: FieldDef<string>, scaleType: ScaleType, size: VgSignalRef) {
```

Comment:
Do you need the whole specifiedAxis,  seems `tickStep` is enough?  Also, maybe move this `tickStep` to last parameter?